doc/dev/dataplane_codegen_quick_start.md

```diff
@@ -0,0 +1,198 @@
+# Getting Started - Generate SDK With Dataplane Codegen
+
+## Prerequisites
+
+- [Python 3.7](https://www.python.org/downloads/windows/) or later is required
+
+- [Nodejs 14.x.x](https://nodejs.org/download/release/latest-v14.x/) or later is required
+
+- [Autorest][https://github.com/Azure/autorest/blob/main/docs/install/readme.md]
+
+## Setup your repo
+
+- Fork and clone the azure-sdk-for-python repo.
+
+- Create a branch to work in.
+
+## Project service name, module name, and package name
+
+Three key pieces of information for your project are the `service_name`, `module_name`, and `package_name`.
+
+The `service_name` is the short name for the Azure service. The `service_name` should match across all the SDK language repos
+and is generally name of the directory in the specification folder of the azure-rest-api-specs repo that contains the REST API definition file.
+
+The `module_name` is the name for the specific feature or function of the REST API that underlies this SDK.
+Usually this name comes from the name of the REST API file itself or one of the directories toward the end of the file path.
+
+In Python, a project's `package name` will normally be `azure-{service_name}-{module_name}`. After release, you can find it in pypi.
```

Comment:
This can be more subtle than "azure-servicename-modulename", for instance "azure-data-tables". or no modulename if the service is single, like "azure-servicebus".
I would say usually, modulename is optional if the service is single, and required if the service has several packages (azure-synapse-accesscontrol).